docs/guidelines.md

```diff
@@ -0,0 +1,10 @@
+# Ionic Guidelines
+#
+#### Readible
+- Code should be intuitive to understand
+- Code should be simple to write. For example capitalization at the beginning of the code is unnecessary

```

Comment:
When you say this I think about user defined classes where it is common to see a capital letter in the beginning.

Something like

```cpp
class Window {}
```

This is defined in the [standard](https://manual.gromacs.org/documentation/5.1-current/dev-manual/naming.html#:~:text=would%20be%20better.-,C%2B%2B%20code,variables).

Is this something you want to stay away from?